<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sales_customers</class>
 <widget class="QWidget" name="sales_customers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>771</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="customer_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_phone">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_province">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>省</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_city">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>市</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_distinct">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>区</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_detail_addr">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>详细地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_name_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_phone_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_province_edit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_city_edit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_distinct_edit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_detail_addr_edit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="customer_sex">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_email">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_postcode">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>邮政编码</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_company">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>公司</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_birthday">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>出生日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="customer_enter_day">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>登记时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_email_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_postcode_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_company_edit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_birthday_edit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_enter_day_edit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="update_customer_info">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>更新信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_customer">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_customer">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_customer">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_sex_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_index">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>编号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
